### PR TITLE
Amls 5293 - Removing bootstrap

### PR DESCRIPTION
**Pull request by slavvco**
Removed bootstrap, upgraded libraries, created some helper classes. Updated all controllers, connectors and views with new imports and/or parameters. Updated specs respectively. Deleted unnecessary configurations.

## Related / Dependant PRs?

- none

## How Has This Been Tested?

- unit tests
- regression

## Checklist

- [x] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/utils/FeatureToggle.scala [hunk 1]
```diff
@@ -14,20 +14,38 @@
  * limitations under the License.
```
Comment (matthewharris1) [inline app/utils/FeatureToggle.scala]: Does this need to be uncommented and looked at / removed?

File: test/connectors/AuthenticatorConnectorSpec.scala [hunk 2]
```diff
@@ -32,24 +36,23 @@ class AuthenticatorConnectorSpec extends AmlsSpec with ScalaFutures {
 
   trait TestFixture {
 
-    val http = mock[HttpPost]
+    val http = mock[HttpClient]
+    val appConfig = mock[AppConfig]
 
-    val featureToggleSetting: Boolean
+//    lazy val app = new GuiceApplicationBuilder()
```
Comment (matthewharris1) [inline test/connectors/AuthenticatorConnectorSpec.scala]: This can be removed?

File: test/services/ConfirmationServiceSpec.scala [hunk 3]
```diff
@@ -149,7 +147,7 @@ class ConfirmationServiceSpec extends PlaySpec
     val activities = mock[BusinessActivities]
     val businessMatching = mock[BusinessMatching]
     val cache = mock[CacheMap]
-    val principle = Principal(None, Accounts(org = Some(OrgAccount("", Org("TestOrgRef")))))
+    //val principle = Principal(None, Accounts(org = Some(OrgAccount("", Org("TestOrgRef")))))
```
Comment (matthewharris1) [inline test/services/ConfirmationServiceSpec.scala]: commented out

File: test/services/LandingServiceSpec.scala [hunk 4]
```diff
@@ -227,7 +226,7 @@ class LandingServiceSpec extends AmlsSpec with ScalaFutures with FutureAwaits wi
         service.desConnector.view(any[String], any())(any[HeaderCarrier], any[ExecutionContext], any[Writes[ViewResponse]])
       } thenReturn Future.successful(viewResponse)
 
-      val user = mock[LoggedInUser]
+      //val user = mock[LoggedInUser]
```
Comment (matthewharris1) [inline test/services/LandingServiceSpec.scala]: Commented out. Below lines in same file aswell

File: app/utils/FeatureToggle.scala (hunk 1 above)
Comment (slavvco) [reply app/utils/FeatureToggle.scala]: done

File: test/connectors/AuthenticatorConnectorSpec.scala (hunk 2 above)
Comment (slavvco) [reply test/connectors/AuthenticatorConnectorSpec.scala]: done

File: test/services/ConfirmationServiceSpec.scala (hunk 3 above)
Comment (slavvco) [reply test/services/ConfirmationServiceSpec.scala]: done

File: test/services/LandingServiceSpec.scala (hunk 4 above)
Comment (slavvco) [reply test/services/LandingServiceSpec.scala]: done

**Comment by ajchilts** [comment]
Plan for this work going forward;

- We already have some phase 3 work in master 
- DES will do their code drop into QA 11/09
- We do not have time to deploy and test this work as a pre-phase 3 release.
- When DES make the drop we would like keep the environment as clean as possible. 
- As this is a large and complex piece that carries risk we should hold off the deploy to master until we know we can get a successful test run in QA.
- If the dev work is complete and the acceptance pack is passing then we should close this ticket as done (dev complete).
- We should then create a new ticket on the backlog to test this piece once we can get into QA with it. 

NOTE: This ticket does remove the old auth library as this was pulled in by frontend bootstrap. However; as we have migrated all of the controllers to new auth there should be no significant risk in leaving this in place in production as we will be making no calls. Auth team will inform us when they do the big switch off in QA and will soon know if we have a problem as we will be in phase 3 E2E. If we have a problem at that point we can discuss a potential hot fix on an issue by issue basis (should we actually encounter any).

**Comment by slavvco** [comment]
Closing as no longer needed, actioned by Play 2.6 upgrade

**Pull request closed without merging**